string/breaking_bad.py

```diff
@@ -16,8 +16,22 @@
 My solution:
 (I sorted the symbols array in descending order of length and ran loop over
 chemicals array to find a symbol match(using indexOf in javascript) which
-worked. But I din't make it through the interview, I am guessing my solution
-was O(n2) and they expected an efficient algorithm.
+worked. But I didn't make it through the interview, I am guessing my solution
+was O(n^2) and they expected an efficient algorithm.
+
+
+note:
+This approach didn't pass interview because it did wrong in conditions like:
+
+input:
+chemicals = ['Amazon', 'Microsoft', 'Google']
+symbols = ['I', 'Am', 'cro', 'Na', 'le', 'abc', 'o']  # add 'o' here
+
+expected:
+['[am]azon', 'mi[cro]soft', 'goog[le]']
+
+exact output:
+['[Am]azon', 'Mi[cro]soft', 'Goog[le]', 'Amaz[o]n', 'Micr[o]s[o]ft', 'G[o][o]gle']
 """
```

Comment:
What is the output that should be printed?
The 'expected' or the 'exact_output'? Any reason for mentioning both?